localplanning.services/src/layouts/Layout.astro

```diff
@@ -0,0 +1,34 @@
+---
+import { ClientRouter } from "astro:transitions";
+import "../styles/global.css";
+import NavBar from "../components/NavBar.astro";
+import Footer from "../components/Footer.astro";
+---
+
+<!doctype html>
+<html lang="en">
+  <head>
+    <ClientRouter />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width" />
+    <link rel="icon" type="image/svg+xml" href="/favicon.ico" />
+    <meta name="generator" content={Astro.generator} />
+    <title>localplanning.services</title>
+  </head>
+  <body class="bg-main">
+    <NavBar />
+    <main>
+      <slot />
```

Comment:
`<slot />` is equivalent to a React `{{ children }}` element. This is where the main content of the site is nested.

Slots can be named, and do not need to be one per-component.

Docs - https://docs.astro.build/en/basics/astro-components/#slots